services/QuillLMS/client/app/bundles/Teacher/components/dashboard/key_metrics.tsx

```diff
@@ -12,7 +13,7 @@ const KeyMetric = ({ number, label, }) => (
     tooltipText={tooltipCopy}
     tooltipTriggerText={(
       <div className="key-metric">
-        <h4>{number}</h4>
+        <h4>{addCommasToThousands(number)}</h4>
```

Comment:
There is a built-in way to do this in Javascript that I would opt for here -- `.toLocaleString()`, as used [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/snapshotCount.tsx#L207).

In general, if we were to need to use that function, I would pull it out of the `Staff` bundle and into `Shared` if we need to use it in multiple bundles, rather than importing it from `Staff` in `Teacher`.